### ISSUE TITLE
Include heading for scsb-file-exists-work-around.md

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/bibdata#2216]
### Steps to recreate
Visit docs/15. scsb-file-exists-work-around.md

### URL
[Access document](https://github.com/pulibrary/bibdata/blob/main/docs/scsb-file-exists-work-around.md)

### Issues
Document does not have heading

### Severity
N/A

### WCAG Conformance level (A, AA, AAA)
N/A

### User impact
Adding a heading facilitates navigation for users who use screen readers.

### Recommended fix
Include heading

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
